src/WebApiToTypeScript/Endpoints/AngularEndpointsService.cs

```diff
@@ -28,22 +28,25 @@ public TypeScriptBlock CreateServiceBlock()
                 .AddStatement($"{Config.ServiceName}.$http = $http;")
                 .AddStatement($"{Config.ServiceName}.$q = $q;", condition: Config.EndpointsSupportCaching)
                 .Parent
-                .AddAndUseBlock("static call<TView>(endpoint: IEndpoint, data)")
-                .AddAndUseBlock($"var call = {Config.ServiceName}.$http<TView>(", isFunctionBlock: true, terminationString: ";")
+                .AddAndUseBlock("static call<TView>(endpoint: IEndpoint, data, httpConfig?: angular.IRequestShortcutConfig)")
+                .AddAndUseBlock("const config = ")
```

Comment:
Extend changes/adds properties, it doesn't assign a new instance of the object therefore const is correct here in my opinion